src/routes/MapExplorer/MapExplorer.connector.js

```diff
@@ -0,0 +1,129 @@
+import { push } from 'connected-react-router'
+import { connect } from 'react-redux'
+import { get, pick } from 'lodash/fp'
+import { FETCH_POSTS_MAP } from 'store/constants'
+import getQuerystringParam from 'store/selectors/getQuerystringParam'
+import presentPost from 'store/presenters/presentPost'
+import getCommunityForCurrentRoute from 'store/selectors/getCommunityForCurrentRoute'
+import getMe from 'store/selectors/getMe'
+import { addQuerystringToPath, baseUrl, personUrl, postUrl } from 'util/navigation'
+
+import {
+  fetchMembers,
+  fetchPosts,
+  storeFetchPostsParam,
+  storeClientFilterParams,
+  getSortedFilteredMembers,
+  getSortedFilteredPosts,
+  getCurrentTopics
+} from './MapExplorer.store.js'
+
+export function presentMember (person, communityId) {
+  return {
+    ...pick([ 'id', 'name', 'avatarUrl', 'locationObject' ], person.ref),
+    type: 'member',
+    community: person.memberships.first()
+      ? person.memberships.first().community.name : null
+  }
+}
+
+export function mapStateToProps (state, props) {
+  const community = getCommunityForCurrentRoute(state, props)
+  const communityId = community && community.id
+  const routeParams = get('match.params', props)
+  const { slug, networkSlug } = routeParams
+
+  const querystringParams = getQuerystringParam(['showDrawer', 't'], null, props)
+
+  var subject
+  if (slug) {
+    subject = 'community'
+  } else if (networkSlug) {
+    subject = 'network'
+  } else {
+    subject = 'all-communities'
+  }
+
+  const fetchMembersParam = {
+    subject,
+    ...pick([
+      'slug',
+      'networkSlug'
+    ], routeParams),
+    ...pick([
+      // TODO: these are actually not being queried by on the server, remove for now?
+      'sortBy',
+      'topic'
+    ], props),
+    boundingBox: state.MapExplorer.fetchPostsParam ? state.MapExplorer.fetchPostsParam.boundingBox : null
+  }
+
+  const fetchPostsParam = {
+    subject,
+    ...pick([
+      'slug',
+      'networkSlug'
+    ], routeParams),
+    ...pick([
+      // TODO: these are actually not being queried by on the server, remove for now?
+      'sortBy',
+      'topic'
+    ], props),
+    boundingBox: state.MapExplorer.fetchPostsParam ? state.MapExplorer.fetchPostsParam.boundingBox : null
+  }
+
+  // TODO: maybe filtering should happen on the presentedPosts? since we do some of that presentation in the filtering code, like calling topics.toModelArray in the filters for every post each time
```

Comment:
I'm always very up for a discussion on selectors and presenters in context of our ReduxORM world. I generally have the tactic of trying to get out of ReduxORM land as quickly as possible at the trunk (so within a selector) vs carrying-over any dependence on it to the component or connector level. Either way, I see nothing objectionable here in particular, and am just wondering what the benefit would be here of doing what you're considering?